web/src/components/AppBar/UserMenu.vue

```diff
@@ -64,7 +63,13 @@ export default {
     },
   },
   methods: {
-    ...mapActions('girder', ['logout']),
+    async logout() {
+      if (toggles.UNIFIED_API) {
+        await publishRest.logout();
+      } else {
+        await girderRest.logout();
+      }
+    },
```

Comment:
Does it makes sense to have a centralized place for this logout logic? Maybe in `rest/index.js`?

Reply:
Maybe. This stuff is going to be rewritten shortly with OAuth though, so I think it's fine to deal with it then.